mnemosyne/mcp_tools.py

```
@@ -739,9 +739,29 @@ def _handle_recall_canonical(arguments: Dict[str, Any]) -> Dict[str, Any]:
+                        "WHERE category=? AND name=? AND valid_until IS NULL LIMIT 1",
+                        (category, name),
+                    )
+                    alt = cur.fetchone()
+                    if alt:
+                        result["hint"] = (
+                            f"Row exists under owner_id '{alt[0]}' but you queried "
+                            f"with '{owner_id}'. Set MNEMOSYNE_DEFAULT_OWNER={alt[0]} "
+                            f"or check your profile/provider config."
+                        )
+            except Exception:
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)